### DISCUSSION TITLE
Can i access Tomorow date inside Query Temaplte

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#3924] [Q&A] [answered]
I have a RefinableDate00 managed property which is linked to a SharePoint site column of type DateOnly, now using the following Query Template, I am able to get the items which have their RefianbleDate00 = today date:-

```
{searchTerms} 
ContentType:"Carrier" 
(RefinableDate00=today)
```

but i also want to build another Search Result web part which show the items which have their RefinableDate00 = Tomorrow. I tried this:-

```
{searchTerms} 
ContentType:"Carrier" 
(RefinableDate00=today+1)
```
but this did not return any items.. any advice ,, how i can get tomorrow date inside Query template?

**Comment by wobba** [comment, marked as answer]
Try this instead: `RefinableDate00>{Today} AND RefinableDate00<{Today+2}`. See https://microsoft-search.github.io/pnp-modern-search/usage/search-results/tokens/#date-tokens

**Reply by mvcsharepointdev** [reply]
@wobba using {today} and {Today+1} will return zero results...
